### PR TITLE
Lo 15 create lead profile

### PR DESCRIPTION
**Pull request by yoyi1073**
Create lead view and store new lead

### CODE REVIEW

File: app/Http/Controllers/CreateProfileController.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        $profile->name = $request->name;
+        $profile->second_name = $request->second_name;
+        $profile->lastname = $request->lastname;
+        $profile->second_lastname = $request->second_lastname;
+        $profile->email = $request->email;
+        $profile->password = $request->password;
+        $profile->phone = $request->phone;
+        $profile->save();
+            return redirect()->route('create-profile.show')
+        ->withSuccess('Se ha registrado satisfactoriamente.');
```
Comment (Diana9218) [inline app/Http/Controllers/CreateProfileController.php]: The arrow of withSuccess, move to right, should be in this way
return redirect()->route('create-profile.show')
        ->withSuccess('Se ha registrado satisfactoriamente.');

File: resources/views/create-profile.blade.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                </div>
+                <br>
+                <div>
+                    <label class="info" for="confirm_email">Confirma correo electrónico:</label>
+                    <input type="text" id="confirm_email" name="confirm_email" placeholder="Confirma correo electrónico*" pattern="[a-z0-5]{8,0" >
+                </div>
+                <br>
+                <div>
+                    <label class="info" for="name">Nombre:</label>
+                    <input type="text" id="name" name="name" placeholder="Primer Nombre*" required>
```
Comment (Diana9218) [inline resources/views/create-profile.blade.php]: The placeholder can be Nombre

File: resources/views/create-profile.blade.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                <div>
+                    <label class="info" for="lastname">Primer apellido:</label>
+                    <input type="text" id="lastname" name="lastname" placeholder="Primer apellido*" required>
+                </div>
+                <br>
+                <div>
+                    <label class="info" for="second_lastname">Segundo apellido:</label>
+                    <input type="text" id="second_lastname" name="second_lastname" placeholder="Segundo apellido*" required>
+                </div>
+                <div>
```
Comment (Diana9218) [inline resources/views/create-profile.blade.php:43]: Should add br

File: resources/views/create-profile.blade.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                    <label class="info" for="phone">Teléfono:</label>
+                    <input type="text" id="phone" name="phone" placeholder="Número de teléfono*" required>
+                </div>
+                <div>
```
Comment (Diana9218) [inline resources/views/create-profile.blade.php:48]: Should add br

File: resources/views/create-profile.blade.php [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                    <label class="info" for="password">Contraseña:</label>
+                    <input type="password" id="password" name="password" placeholder="Contraseña*" required>
+                </div>
+
+            </fieldset>
+        </section>
+        <br>
+        <div>
+            <input id="privacy" type="checkbox" name="privacy" checked>
```
Comment (Diana9218) [inline resources/views/create-profile.blade.php]: This input is not checked, quit off this attribute

File: app/Http/Controllers/CreateProfileController.php (hunk 1 above)
Comment (yoyi1073) [reply app/Http/Controllers/CreateProfileController.php]: Done

**Review by Diana9218 — APPROVED** [review]

File: app/Http/Controllers/CreateProfileController.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+use App\Models\Lead;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class CreateProfileController extends Controller
+{
+    public function show(): View
+    {
+        return view('/create-profile');
```
Comment (zahirolamadrid) [inline app/Http/Controllers/CreateProfileController.php]: It's not necessary to use slash when you return a view

File: resources/views/create-profile.blade.php [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+        <br>
+        Son necesarios para continuar</p>
+    <form method="POST" action={{route('create-profile.store')}}>
+        @csrf
+        <section>
+            <fieldset>
+                <br>
+                <div>
+                    <label class="info" for="email">Correo electrónico:</label>
+                    <input type="text" id="email" name="email" placeholder="Correo electrónico*" pattern="[a-z0-5]{8,0" required>
```
Comment (zahirolamadrid) [inline resources/views/create-profile.blade.php]: This pattern is unnecessary

File: resources/views/create-profile.blade.php [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+                </div>
+                <br>
+                <div>
+                    <label class="info" for="confirm_email">Confirma correo electrónico:</label>
+                    <input type="text" id="confirm_email" name="confirm_email" placeholder="Confirma correo electrónico*" pattern="[a-z0-5]{8,0" >
```
Comment (zahirolamadrid) [inline resources/views/create-profile.blade.php]: This pattern is unnecessary

File: resources/views/create-profile.blade.php [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+                </div>
+                <br>
+                <div>
+                    <label class="info" for="name">Nombre:</label>
+                    <input type="text" id="name" name="name" placeholder="Nombre*" required>
```
Comment (zahirolamadrid) [inline resources/views/create-profile.blade.php]: It's better without asterisks

File: resources/views/create-profile.blade.php [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+                <div>
+                    <label class="info" for="password">Contraseña:</label>
+                    <input type="password" id="password" name="password" placeholder="Contraseña*" required>
+                </div>
+
+            </fieldset>
+        </section>
+        <br>
+        <div>
+            <input id="privacy" type="checkbox" name="privacy">
```
Comment (zahirolamadrid) [inline resources/views/create-profile.blade.php]: This should be required or removed

**Review by zahirolamadrid — APPROVED** [review]

**Pull request merged**